### PR TITLE
test_all_sandia: updates

### PR DESCRIPTION
**Pull request by ndellingwood**
[skip ci]

### CODE REVIEW

File: scripts/testing_scripts/test_all_sandia [hunk 1]
```diff
@@ -845,8 +847,12 @@ single_build_and_test() {
     echo "kokkos devices: ${LOCAL_KOKKOS_DEVICES}"
     echo "kokkos cxx: ${cxxflags}"
 
+    # KOKKOS_OPTIONS and KOKKOS_CUDA_OPTIONS are exported and detected by kokkos' generate_makefile.sh during install of kokkos; we pass them to the reproducer script instructions
     echo "  #   Use generate_makefile line below to call cmake which generates makefile for this build:" &> call_generate_makefile.sh
-    echo "        ${KOKKOS_PATH}/generate_makefile.bash --with-devices=$LOCAL_KOKKOS_DEVICES $ARCH_FLAG --compiler=$(which $compiler_exe) --cxxflags=\"$cxxflags\" --cxxstandard=\"$cxx_standard\" --ldflags=\"$ldflags\" $CUDA_ENABLE_CMD --kokkos-path=${KOKKOS_PATH} --no-examples $extra_args" &>> call_generate_makefile.sh
+    echo "        ${KOKKOS_PATH}/generate_makefile.bash --with-devices=$LOCAL_KOKKOS_DEVICES $ARCH_FLAG --compiler=$(which $compiler_exe) --cxxflags=\"$cxxflags\" --cxxstandard=\"$cxx_standard\" --ldflags=\"$ldflags\" $CUDA_ENABLE_CMD --kokkos-path=${KOKKOS_PATH} --with-options=${KOKKOS_OPTIONS} --with-cuda-options=${KOKKOS_CUDA_OPTIONS}--no-examples $extra_args" &>> call_generate_makefile.sh
```
Comment (jeffmiles63) [inline scripts/testing_scripts/test_all_sandia:852]:
```suggestion
    echo "        ${KOKKOS_PATH}/generate_makefile.bash --with-devices=$LOCAL_KOKKOS_DEVICES $ARCH_FLAG --compiler=$(which $compiler_exe) --cxxflags=\"$cxxflags\" --cxxstandard=\"$cxx_standard\" --ldflags=\"$ldflags\" $CUDA_ENABLE_CMD --kokkos-path=${KOKKOS_PATH} --with-options=${KOKKOS_OPTIONS} --with-cuda-options=${KOKKOS_CUDA_OPTIONS} --no-examples $extra_args" &>> call_generate_makefile.sh
```

Comment (ndellingwood) [reply scripts/testing_scripts/test_all_sandia:852]: @jeffmiles63 good catch, just pushed the fix

Comment (ndellingwood) [reply scripts/testing_scripts/test_all_sandia]: Oop, should be fixed now, I didn't add the change properly to my previous amended commit

**Comment by dalg24** [comment]
Trying to close and re-open to re-trigger Travis

**Comment by dalg24** [comment]
Retest this please

**Comment by dalg24** [comment]
Oh now I see the [skip ci].  Nevermind.

**Pull request merged**